change/@ni-nimble-components-52d4fa5f-b81b-4794-9e89-357a43bfb7de.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Migrate to @ni/fast-* packages",
+  "packageName": "@ni/nimble-components",
+  "email": "rajsite@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

Comment:
Thoughts on making this a major change for Blazor too? I think we could achieve that by setting `"dependentChangeType": "major"`.

Reply:
Makes sense to me! Updated